### PR TITLE
Implement zigsay applet

### LINKED ISSUE #54: Applet Suggestion: zigsay
zigsay would be a program that generates ASCII pictures of a ziguana (or other image to be decided upon) with a message

### PR DESCRIPTION
**Pull request by gabrielsimoes**
Closes #54.

### CODE REVIEW

**Comment by AndroidKitKat** [comment]
poggers

**Comment by sambattalio** [comment]
As mentioned a little in the irc channel, if you wouldn't mind switching from the testing allocator and using the opt.zig we would happily merge this. A good example is basename.zig which uses the opt struct to parse args. This looks pretty sweet man, really appreciate the PR :D.

File: src/zigsay.zig [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    \\            ( `-.__.-' )  ( (    .
+    \\   .         \        /    \ \
+    \\       .      \      /      ) )        .
+    \\            .' -.__.- `.-.-'_.'
+    \\ .        .'  /-____-\  `.-'       .
+    \\          \  /-.____.-\  /-.
+    \\           \ \`-.__.-'/ /\|\|           .
+    \\          .'  `.    .'  `.
+    \\          |/\/\|    |/\/\|
+    \\jro
```
Comment (sambattalio) [inline src/zigsay.zig:24]: Just wondering, what is the jro?

**Comment by sambattalio** [comment]
This looks like it is working great! Just check out the one comment from review and i think its ready to roll :D

File: src/zigsay.zig (hunk 1 above)
Comment (gabrielsimoes) [reply src/zigsay.zig:24]: Credits to the author of the drawing, taken from [asciiart](https://www.asciiart.eu/animals/reptiles/lizards).

Comment (sambattalio) [reply src/zigsay.zig:24]: perfect!

**Pull request merged**